src/vellum/workflows/nodes/displayable/subworkflow_deployment_node/node.py

```diff
@@ -105,41 +125,112 @@ def run(self) -> Iterator[BaseOutput]:
         for event in subworkflow_stream:
             if event.type != "WORKFLOW":
                 continue
+            # Always emit the appropriate workflow event
             if event.data.state == "INITIATED":
-                continue
+                self._context._emit_subworkflow_event(
+                    WorkflowExecutionInitiatedEvent(
+                        span_id=event.execution_id,
+                        trace_id=event.run_id,
+                        name="workflow.execution.initiated",
+                        body=WorkflowExecutionInitiatedBody(
+                            inputs=BaseInputs(
+                                **{value.name: cast(value.type, value.value) for value in event.data.inputs}
+                            ),
+                            # We need the workflow definition, or we make this an optional field
+                            workflow_definition=WorkflowType.__class__,
```

Comment:
@dvargas92495 last piece here, besides test, is what to do about workflow_definition and parent fields

I think map should be fine - but linking this to a WIP PR here: https://github.com/vellum-ai/vellum/pull/6683

other option is asking vembda what the resource is -- or make a generic workflowbase that is a subclass of the base class